wgpu-core/src/command/render.rs

```diff
@@ -427,6 +446,15 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         // instead of the special read-only one, which would be `None`.
         let mut is_ds_read_only = false;
 
+        struct OutputAttachment<'a> {
+            texture_id: &'a Stored<id::TextureId>,
+            range: &'a hal::image::SubresourceRange,
+            previous_use: Option<TextureUse>,
+            new_use: TextureUse,
+        }
+        const MAX_TOTAL_ATTACHMENTS: usize = MAX_COLOR_TARGETS + MAX_COLOR_TARGETS + 1;
```

Comment:
Either this is a typo or should be `2 * MAX_COLOR_TARGETS + 1`

Reply:
That's intentional :)
I don't mind changing it to what you suggest

Reply:
Alright :) It just looked like it could have been a typo for COLOR + DEPTH_STENCIL or whatever